lib/rules/function-comma-newline-after/index.js

```diff
@@ -32,7 +33,19 @@ const rule = function(expectation) {
       root,
       result,
       locationChecker: checker.afterOneOnly,
-      checkedRuleName: ruleName
+      checkedRuleName: ruleName,
+      fix: context.fix
+        ? (div, index, nodes) => {
+            return fixer({
+              div,
+              index,
+              nodes,
+              expectation,
+              position: "after",
+              symb: "\n"
```

Comment:
Please use `context.newline` instead of `\n`.

Also, could you add a test case that uses CRLF?